### ISSUE TITLE
IA: Interview key stakeholders Part II

### ISSUE THREAD

**Issue by kristinashu** [mozilla/foundation.mozilla.org#4228]
First round of interviews were done in #4112 and stakeholders were identified in #4167 

Report on this interviews will be done in - IA: Report on key stakeholders interviews Part II #4229.

### Interviews
**DONE**
- [x] Schedule interview with Jon
- [x] Run interview + card sorting activity with Jon
- [x] Schedule interview with Juan
- [x] Run interview + card sorting activity with Juan
- [x] Schedule interview with Stephanie L. 
- [x] Run interview + card sorting activity with Stephanie L
- [x] Schedule interview with Amy
- [x] Run interview + card sorting activity with Amy
- [x] Schedule interview with Sarah Swayze
- [x] Run interview + card sorting activity with Sarah Swayze
- [x] Schedule interview with Abdoul
- [x] Run interview + card sorting activity with Abdoul

**REMAINING**
- [x] Schedule interview with Ashley 
- [x] Run interview + card sorting activity with Ashley
- [ ] Schedule interview with Bob - MAR 16 - out of sprint cycle
- [ ] Run interview + card sorting activity with Bob

**WAITING ON REPLY**
- [ ] Schedule interview with Anil - EMAIL SENT
- [ ] Run interview + card sorting activity with Anil

- Taís' note: @xmatthewx will sit on those interviews (the ones marked as "EMAIL SENT"). Matthew, I will mark you as optional in the meeting invite so you can decide which ones you want to attend.

- @taisdesouzalessa to add the rest of the names please!

cc @beccaklam @sabrinang

**Comment by kristinashu** [comment]
@taisdesouzalessa @sabrinang thank you for all the amazing interviews you've done! Tais, could you please open a new ticket for next sprint for any remaining work and then close this one?

**Comment by taisdesouzalessa** [comment]
Only one interview left. Ticket is here: #4322
